src/components/Common/SearchInput.tsx

```diff
@@ -45,21 +46,12 @@ const SearchInput = ({ isLoggedIn, className }: InputProps) => {
           onBlur={() => setSearchBar('')}
         />
         {searchBar && (
-          <MdClear
-            className='absolute right-2 top-0 bottom-0 my-auto text-2xl text-gray-600 cursor-pointer z-10'
-            onClick={handleSearchBarClear}
-          />
+          <div className='absolute right-2 top-0 bottom-0 flex items-center gap-2 text-gray-600'>
+            <span className=''>{12 - searchBar.length}</span>
```

Comment:
```suggestion
            <span>{12 - searchBar.length}</span>
```

className 적용할 거 없으면 제거해도 될 것 같아요!